### ISSUE TITLE
[Launch] BDD: Does BDD Actually Work?

### ISSUE THREAD

**Issue by luke-gcio** [department-of-veterans-affairs/va.gov-team#11812]
## Issue Description
Note: quick review of JSON from Michael Harlow to make sure we are asking all the necessary questions.

Story to test/integrate and make sure we actually get a response back from EVSS when we submit a BDD claim.

1. 90-180 (Real BDD)
2. 0-89 (Pre-OG Claim should still be accepted by EVSS)

## Acceptance Criteria
- [ ] Both of the above scenarios work
- [ ] Stories to fix broken things written

## How to configure this issue
- [x] **Attached to a Milestone** (when will this be completed?)
- [x] **Attached to an Epic** (what body of work is this a part of?)
- [x] **Labeled with Team** (`product support`, `analytics-insights`, `operations`, `service-design`, `tools-be`, `tools-fe`)
- [x] **Labeled with Practice Area** (`backend`, `frontend`, `devops`, `design`, `research`, `product`, `ia`, `qa`, `analytics`, `contact center`, `research`, `accessibility`, `content`)
- [x] **Labeled with Type** (`bug`, `request`, `discovery`, `documentation`, etc.)

**Comment by ndsprinkle** [comment]
First test confirms we need to ask "Is VA Employee" question. Will add this in #11858

**Comment by annaswims** [comment]
staging test users linked in https://github.com/department-of-veterans-affairs/va.gov-team/issues/11892

**Comment by ndsprinkle** [comment]
I was able to successfully submit a BDD claim, so we are at least making past the schema validation now, but will need to see how things go upstream. This was using user 229, so I will also need to see how things go with the new staging users.

**Comment by ndsprinkle** [comment]
Attempted to submit a BDD claim with user 250 and the form did successfully submit like before, but still not seeing anything in the CST. Will investigate further, jobId: 8f3181bb7d5d0925a3b6e6ea

**Comment by luke-gcio** [comment]
Still working through integration issues, moving to sprint 28.

**Comment by ndsprinkle** [comment]
I would say the conclusion to this ticket is...yes! We were able to submit a few forms using a BDD separation date (90-180 days from the current date) that successfully hit the EVSS endpoint and ultimately appeared in the claim status tool. There still should be some more testing done for early 526 submissions (separation date that is 1-89 days from current date) as the ones that we tested failed because of invalid data, but that situation is not technically a BDD submission, so I'm not sure if it counts toward this ticket.
